packages/rum/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -451,8 +460,11 @@ describe('rum view measures', () => {
 
     setupBuilder = setup()
       .withFakeLocation('/foo')
-      .withViewCollection()
-      .beforeBuild((lifeCycle) => lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler))
+      .beforeBuild(({ location, lifeCycle }) => {
+        lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler)
+        const { stop } = trackViews(location, lifeCycle)
+        return stop
+      })
```

Comment:
For the cleanup, I think could be more explicit or more implicit.

explicit: remove the logic from the helper and it makes more clear what we want to do with stop in the test
```js
      beforeBuild(({ location, lifeCycle, cleanupTasks }) => {
        lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler)
        const { stop } = trackViews(location, lifeCycle)
        cleanupTasks.push(stop)
      })
```

implicit: handle by default returned object with stop since it is seems to be what we mostly use
```js
      beforeBuild(({ location, lifeCycle, cleanupTasks }) => {
        lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler)
        return trackViews(location, lifeCycle)
      })
```

no strong opinion on it, wdyt?

Reply:
Returning an object with `stop` looks good to me